packages/mui-joy/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/joy",
-  "version": "5.0.0-beta.46",
+  "version": "5.0.0-beta.47",
```

Comment:
Latest published version of joy is `5.0.0-beta.48` (released from the v5.x branch), so we should bump it to 49.

Alternatively, we could release it from the v5.x branch, what do you think @aarongarciah?

Reply:
We'll also need a base release 😅 @mj12albert

Reply:
I can't recall any issues.